### ISSUE TITLE
MacOS compatibility? 

### ISSUE THREAD

**Issue by Awesomedonut** [Purfview/whisper-standalone-win#368]
I read in the README that this solution is compatible with Mac. I went to the releases tab and there aren't any mac executables. I downloaded the one for Linux, since it's kinda close to Mac, but it wasn't compatible and didn't run. 

Just for fun, I downloaded one of the earlier releases (Whisper-Faster_r186.1_macOS-x86-64) but the code didn't seem to run (I just waited on CLI for a long time and nothing visible happened). In the releases it says it was depreciated, so yeah it makes sense

Basically, I was just wondering how to run this solution on MacOS. Is there some documentation somewhere I'm not reading?

**Comment by Purfview** [comment]
>Just for fun, I downloaded one of the earlier releases (Whisper-Faster_r186.1_macOS-x86-64) but the code didn't seem to run (I just waited on CLI for a long time and nothing visible happened)

It should be compatible with macOS v10.15 and above on x86-64 CPUs, if it didn't show any error them I guess mac is doing its mac things and you need to wait.
There's no other releases for macOS.
